### PR TITLE
Use app specific log folder

### PR DESCRIPTION
**Pull request by tobiasKaminsky**

### CODE REVIEW

File: src/com/owncloud/android/lib/common/utils/Log_OC.java [hunk 1]
```diff
@@ -73,30 +74,26 @@ public static void wtf(String TAG, String message) {
...
-
+        try {
             // Create the current log file if does not exist
-            mLogFile.createNewFile();
+            isEnabled = mLogFile.createNewFile();
             mBuf = new BufferedWriter(new FileWriter(mLogFile, true));
-            isEnabled = true;
 
-            if (isFileCreated) {
+            if (isFolderCreated && isEnabled) {
```
Comment (AndyScherzinger) [inline src/com/owncloud/android/lib/common/utils/Log_OC.java]: What if the folder already exists?

Comment (tobiasKaminsky) [inline src/com/owncloud/android/lib/common/utils/Log_OC.java]: Should not happen as this is the app specific folder which will be deleted when the app is removed.
But I have added it, just to be safe.

**Comment by AndyScherzinger** [comment]
👍 looks good to me

**Comment by mario** [comment]
👍

**Pull request merged**